README.md

```
@@ -2,345 +2,134 @@
 
-Grab the latest DMG from [GitHub Releases](https://github.com/Octane0411/open-vibe-island/releases) — signed and notarized, ready to run.
+### 即时终端识别
+原版依赖 2 秒一次的轮询来发现新终端。增强版在收到 Hook 事件时**立即触发**进程扫描和终端匹配，新开的终端运行 AI agent 后**瞬间**出现在面板中。
 
-### Option 2: Build from source
-
-```bash
-git clone https://github.com/Octane0411/open-vibe-island.git
-cd open-vibe-island
-open Package.swift   # Opens in Xcode — hit Run
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)